src/inmanta/data/dataview.py

```diff
@@ -1332,4 +1348,5 @@ def filter(self, filter_statements: list[str], values: list[object]) -> "Prelude
             db_order=self.db_order,
             backward_paging=self.backward_paging,
             prelude_query_builder=self._prelude_query_builder.filter(filter_statements, values),
+            prelude_query_builder_extra=self._prelude_query_builder_extra,
```

Comment:
Here I think I need to do something (as for the prelude_query_builder) but I don't understand why we need to apply a limit/order on the subqueries, I would have expected doing it only on ht main query was enough